packages/turf-constrained-tin/bench.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+const Benchmark = require('benchmark');
+const tin = require('@turf/tin').default;
+const constrainedTin = require('./');
+
+const points = JSON.parse(fs.readFileSync(path.join(__dirname, 'test', 'Points.json')));
+
+/**
+ * Benchmark Results
+ *
+ * constrained Tin (cdt-js) with 1 edge: 5.494ms
```

Comment:
This result is soooo slower compares to the others. Is it correct?

Reply:
I don't know the reason, but it is the first time of calling the function in this benchmark, so I think maybe there are some initializations...

I add dummy function calls of both turf-constrained-tin and turf-tin before testing, it becomes better.

constrained Tin (cdt-js) with 1 edge: 1.398ms
constrained Tin (cdt-js) with 2 edges: 0.973ms
constrained Tin (cdt-js) with 3 edges: 1.915ms
normal Tin (as reference, no constrained): 0.359ms
constrained Tin (cdt-js) with 1 edge x 21,375 ops/sec ±3.26% (80 runs sampled)
constrained Tin (cdt-js) with 2 edges x 23,076 ops/sec ±4.29% (75 runs sampled)
constrained Tin (cdt-js) with 3 edges x 23,855 ops/sec ±1.70% (81 runs sampled)
normal Tin (as reference, no constrained) x 21,021 ops/sec ±1.56% (83 runs sampled)